### ISSUE TITLE
[🕹️] Define group and assign to data room

### ISSUE THREAD

**Issue by ykburda** [mfts/papermark#646]
### What side quest or challenge are you solving?

Feature request

### Points

?

### Description

It is only possible define group members one datarooms. This is somehow hard to follow. We should also need define team specific groups generally and assign to dataroom. Eg: generally defined A group with members , B group, C group and go to specific dataroom and assaigning .

### Provide proof that you've completed the task

?

**Comment by mfts** [comment]
We were thinking about this at first, but then after talking with customers. Typically their groups are focused around single datarooms. That gives them more control.

We keep that in the back of our head to elevate the groups to the team level. 

Thanks for the impulse :)
